packages/amplify-python-function-runtime-provider/src/util/invokeUtil.ts

```diff
@@ -20,7 +20,10 @@ export async function pythonInvoke(context: any, request: InvocationRequest): Pr
     throw new Error(`Could not find 'python3' or 'python' executable in the PATH.`);
   }
 
-  const childProcess = execa('pipenv', ['run', pyBinary, shimPath, handlerFile + '.py', handlerName]);
+  const childProcess = execa('pipenv', ['run', pyBinary, shimPath, handlerFile + '.py', handlerName], {
```

Comment:
nit: file name is 'invokeUtils', but it was changed to 'invoke' in other runtime provider

Reply:
gonna leave it because the other files in the python package are named this way (buildUtils, packageUtils, etc)